ironfish/src/wallet/wallet.ts

```diff
@@ -1424,7 +1424,7 @@ export class Wallet {
       const multisigSecret = await this.walletDb.getMultisigSecret(
         Buffer.from(accountValue.multisigKeys.identity, 'hex'),
       )
-      if (!multisigSecret) {
+      if (!multisigSecret || !multisigSecret.secret) {
```

Comment:
since we do this check in a couple of places it might be worthwhile to keep a method `getMultisigSecret` that returns just the secret